### ISSUE TITLE
FEATURE_BROWSER_EMULATION not in winreg.HKEY_CURRENT_USER

### ISSUE THREAD

**Issue by frip** [r0x0r/pywebview#6]
Running Windows 7, Python 2.7:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Fritz\Documents\webgui-demo\venv\lib\site-packages\webview\main.py", line 72, in create_window
    gui.create_window(_make_unicode(title), _transform_url(url), width, height, resizable, fullscreen)
  File "C:\Users\Fritz\Documents\webgui-demo\venv\lib\site-packages\webview\win32.py", line 185, in create_window
    set_ie_mode()
  File "C:\Users\Fritz\Documents\webgui-demo\venv\lib\site-packages\webview\win32.py", line 239, in set_ie_mode
    0, winreg.KEY_ALL_ACCESS)
WindowsError: [Error 2] The system cannot find the file specified
```

It seems that on my machine the FEATURE_BROWSER_EMULATION is in HKEY_LOCAL_MACHINE instead of HKEY_CURRENT_USER. When I change the `winreg.OpenKey(...)` accordingly I get an \[Error 5] (Access denied). :-(

**Comment by r0x0r** [comment]
Do you run it as an administrator or a normal user?

**Comment by frip** [comment]
I ran it as a normal user.
After looking at another PC where everything works I think it would be good if FEATURE_BROWSER_EMULATION would be created new for HKEY_CURRENT_USER if it does not exist yet.
I haven't finished trying it myself, but I found something else along the way: FileNotFoundError doesn't exist in Python2 (cf. e.g. http://stackoverflow.com/questions/21367320/) - it is used now in `get_ie_mode()`.

**Comment by frip** [comment]
This seems to work for me:
```
    try:
        browser_emulation = winreg.OpenKey(winreg.HKEY_CURRENT_USER,
                                           r'Software\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BROWSER_EMULATION',
                                           0, winreg.KEY_ALL_ACCESS)
    except WindowsError:
        browser_emulation = winreg.CreateKeyEx(winreg.HKEY_CURRENT_USER,
                                             r'Software\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BROWSER_EMULATION',
                                             0, winreg.KEY_ALL_ACCESS)
```

**Comment by r0x0r** [comment]
Thanks for the fix. I will incorporate it in the next version, as soon as I get back from travelling.

**Comment by r0x0r** [comment]
Fix incorporated in 0.8
